corehq/apps/accounting/templates/accounting/partials/confirm_plan_summary.html

```diff
@@ -33,23 +25,21 @@ <h4>
         {% endfor %}
       </dl>
     </div>
-    <p class="plan-desc">
-      {{ plan.description }}
-    </p>
+    <p class="plan-desc">{{ plan.description }}</p>
   </div>
 </div>
 
-<div class="alert {% if is_same_edition %}alert-info{% else %}alert-warning{% endif %} alert-subscription">
+<div
+  class="alert {% if is_same_edition %}alert-info{% else %}alert-warning{% endif %} alert-subscription"
```

Comment:
```
class="
  alert alert-subscription
  {% if is_same_edition %}
    alert-info
  {% else %}
    alert-warning
  {% endif %}
"
```
feels even more readable tbh
or at least if/else on one separate line

but definitely grouping the main classes together in the beginning is important

Also, what does `alert-subscription` do? is it js or a custom class? if `js` please add `js-` prefix

Reply:
It's a custom styling class defined here https://github.com/dimagi/commcare-hq/blob/e60d3cf73a31e849e8434e1b790e3ae6846bf3a6/corehq/apps/hqwebapp/static/accounting/less/pricing-main.less#L347-L350

Updated the look of this classname here https://github.com/dimagi/commcare-hq/pull/36448/commits/f858955d5a1303be5be6a37be25c5ca667877046